config/manifests/bases/idp-mgmt-operator.clusterserviceversion.yaml

```diff
@@ -46,4 +46,4 @@ spec:
   provider:
     name: Red Hat
     url: https://github.com/identitatem
-  version: 0.1.0
+  version: 0.1.1
```

Comment:
We should have a replaces or similar in here for 0.1.0 as well so that OLM knows how to upgrade from one version to the next. 

If you don't care about going "through" all versions to upgrade (i.e. for database migrations), you can also use skips/skipRanges.

Reply:
I am new to CSV, let me find doc so I know what you are referring to

Reply:
So is this what you want added?
replaces: idp-mgmt-operator.v0.1.0

Reply:
Yep, this looks good to me.